### PR TITLE
Avoid triggering click event on key down, and set focus on search input after submit

### PR DESCRIPTION
**Pull request by hrojek**
This PR fixes the behavior of triggering the request on any key down, implementing form submit behavior. Additionally, sets the focus on the search input after submitting. Implemented in Retrieval, Reading and QA.

### CODE REVIEW

File: src/applications/qa/index.js [hunk 1]
```diff
@@ -133,10 +135,32 @@ function QuestionAnswering({ application, showSettings }) {
...
+                retrievers.selectedRetriever,
+                selectedCollection,
+                readers.selectedReader,
+                setAnswers,
+                setProcessing,
+                setProcessed,
+                dispatch
+              );
+
+              searchRef.current.focus();
```
Comment (seirasto) [inline src/applications/qa/index.js:157]: Can you add a comment here "Switch focus to question text field"

Comment (hrojek) [reply src/applications/qa/index.js:159]: Sure. Comments added.

**Review by seirasto — APPROVED** [review]
LGTM

File: Dockerfile [hunk 2]
```diff
@@ -43,7 +43,14 @@ CMD ["nginx", "-g", "daemon off;"]
 
 # needed for chmod
 USER root
+
 # needed for write access for deployment.sh
 RUN mkdir /tmp/proxy_temp && chmod 0757 /usr/share/nginx/html/
+
```
Comment (seirasto) [inline Dockerfile:49]: what is this for?

Comment (hrojek) [reply Dockerfile:49]: There's a mismatch between the feature branch's base and the PR base. These changes are from acl2023_demo.

**Comment by hrojek** [comment]
Closed due to base branch mismatch.

**Pull request closed without merging**